python-avd/pyavd/_eos_designs/structured_config/network_services/router_bgp.py

```diff
@@ -276,15 +276,17 @@ def _update_router_bgp_vrf_evpn_or_mpls_cfg(
         route_targets = {"import": [], "export": []}
```

Comment:
Actually, looking into this. we can refactor this as we don't really need to do the _from_dict anymore @MaheshGSLAB.

Just append directly to the `bgp_vrf.route_targets.import` / `bgp_vrf.route_targets.export` wherever needed